canonical-web-and-design/charmhub.io
# Update feature list

## Done

Update list of feature charms following latest import

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
- Make sure feature list is well updated

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/3306
